src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Repository/LocationMetaRepository.cs

```diff
@@ -153,16 +146,58 @@ FROM read_csv('{dataSetVersionPathResolver.CsvDataPath(dataSetVersion):raw}', AL
         }
     }
 
+    private async Task<List<LocationOptionMetaRow>> GetLocationOptionMetas(
```

Comment:
As per our conversation about the naming of a similar method in `FilterMetaRepository`, if we go with a more explicit naming of `GetFilterMetasFromCsv`, we should adjust the naming in this repository as well.

The same could be done for `TimePeriodMetaRepository`, `GeographicLevelMetaRepository` as well.

Reply:
I went for the simple "Get" in the end, so leaving this be :)